OptimizelySDK.Tests/EntityTests/ExperimentTest.cs

```diff
@@ -0,0 +1,56 @@
+﻿/* 
+ * Copyright 2019, Optimizely
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using NUnit.Framework;
+
+namespace OptimizelySDK.Tests.EntityTests
+{
+    [TestFixture]
+    public class ExperimentTest
+    {
+        private ProjectConfig Config;
+
+        [TestFixtureSetUp]
+        public void Setup()
+        {
+            Config = ProjectConfig.Create(TestData.TypedAudienceDatafile, null, null);
+        }
+
+        [TestFixtureTearDown]
+        public void Cleanup()
+        {
+            Config = null;
+        }
+
+        #region GetAudienceConditionsOrIds Tests
+
+        [Test]
+        public void TestGetAudienceConditionsOrIdsRetrieveAudienceIdsIfConditionsNotExist()
```

Comment:
Can we also add a testcase with audienceConditions set to empty array?
It should return empty array instead of audienceIds

Reply:
Added this scenario in TestGetAudienceConditionsOrIdsRetrieveAudienceConditionsIfExist test.